test/configuration_policy_prune.go

```diff
@@ -29,41 +31,84 @@ func ConfigPruneBehavior(labels ...string) bool {
 			Expect(err).To(BeNil())
 
 			Eventually(func(g Gomega) {
-				outManaged, err := OcManaged(
+				outManaged, err := OcHosting(
 					"get", "configurationpolicies", "-A",
 				)
 				GinkgoWriter.Printf("cleanPolicy OcManaged configurationpolicy output: %v\n", outManaged)
 				g.Expect(outManaged).To(BeEmpty())
 				g.Expect(err).To(BeNil())
 			}, DefaultTimeoutSeconds, 1).Should(Succeed())
 
-			outManaged, err := OcManaged(
-				"delete", "events", "-n", ClusterNamespace,
-				"--field-selector=involvedObject.name="+UserNamespace+"."+policyName,
-				"--ignore-not-found",
-			)
-			GinkgoWriter.Printf("cleanPolicy OcManaged policy event output: %v\n", outManaged)
-			Expect(err).To(BeNil())
-			outManaged, err = OcManaged(
-				"delete", "events", "-n", ClusterNamespace,
-				"--field-selector=involvedObject.name="+policyName,
-				"--ignore-not-found",
+			By("Removing any finalizers from the configmap")
+
+			_, _ = OcManaged("patch", "configmap", pruneConfigMapName, "-n", "default",
+				"--type=json", "-p", `[{"op":"remove", "path":"/metadata/finalizers"}]`)
```

Comment:
Why are these changes needed to the prune test?

Reply:
in hosted mode, deleting events interrupts testings I guess it has event latency or something

Reply:
What do you mean by interrupts testing? The tests should only run serially.